src/Command/IndexService/BootstrapCommand.php

```diff
@@ -22,13 +22,19 @@
 use Pimcore\Console\Traits\Timeout;
 use Pimcore\Model\DataObject;
 use Pimcore\Model\DataObject\Listing;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * @internal
  */
+#[AsCommand(
+    name: 'ecommerce:index:create-or-update-indices',
+    description: 'Bootstrap tasks creating/updating index (for all tenants), use one of the options --create-or-update-index-structure or --update-index',
+    aliases: ['ecommerce:indexservice:bootstrap']
+)]
 class BootstrapCommand extends AbstractIndexServiceCommand
```

Comment:
just leaving a placeholder to remind that class name `BoostrapCommand` and command `ecommerce:indexservice:bootstrap` have changed, may require a filename and class name change too.